physionet-django/environment/templates/environment/workspace_being_provisioned.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'base.html' %}
+
+{% load static %}
+
+{% load project_templatetags %}
+
+{% block title %}Research environments{% endblock %}
+
+{% block local_css %}
+<link rel="stylesheet" type="text/css" href="{% static 'environment/css/environment-base.css' %}">
+{% endblock %}
+
+{% block local_js_top %}
+<script src="{% static 'custom/js/cookie.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div class="container">
+  {% include "message_snippet.html" %}
+  <div class="jumbotron">
+    <h1>Research Environments Home</h1>
```

Comment:
We could also extract a base template from this, e.g. `base_environment_home` that would have the `jumbotron` title and `lead`, all the css/js and a `content` block or the actual specific templates